### PR TITLE
test(generators): Add test case for production CSS config

### PR DESCRIPTION
**Pull request by jamesgeorge007**
**What kind of change does this PR introduce?**
test update

**Did you add tests for your changes?**
Yep

**If relevant, did you update the documentation?**
Nope

**Summary**
Follow up of #1471

**Does this PR introduce a breaking change?**
Nope

**Other information**
N/A

### CODE REVIEW

File: packages/generators/__tests__/utils/styleSupport.test.ts [hunk 1]
```diff
@@ -22,18 +22,27 @@ describe('styleSupport', () => {
...
+        gen.isProd = true;
+        const { ExtractUseProps, regExpForStyles } = style(gen, StylingType.CSS);
+        expect(gen.dependencies).toEqual(['css-loader']);
+        expect(regExpForStyles).toEqual('/.css$/');
+        expect(ExtractUseProps.length).toEqual(1);
+    });
+
     it('generates SASS configuration', () => {
         const gen = getMockGenerator();
         const { ExtractUseProps, regExpForStyles } = style(gen, StylingType.SASS);
```
Comment (evilebottnawi) [inline packages/generators/__tests__/utils/styleSupport.test.ts:36]: Can we fix a warning?

Comment (jamesgeorge007) [reply packages/generators/__tests__/utils/styleSupport.test.ts:36]: Can you elaborate?

Comment (snitin315) [reply packages/generators/__tests__/utils/styleSupport.test.ts:36]: > 'regExpForStyles' is assigned a value but never used

this warning I guess.

Comment (snitin315) [inline packages/generators/__tests__/utils/styleSupport.test.ts:36]:
```suggestion
        const { ExtractUseProps } = style(gen, StylingType.SASS);
```

Comment (jamesgeorge007) [reply packages/generators/__tests__/utils/styleSupport.test.ts:36]: It is required.

**Comment by evilebottnawi** [comment]
Thanks!

**Pull request merged**